docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc

```diff
@@ -20,36 +20,49 @@ There are four steps to set up the Time Series DB for your system:
 . Obtain an activation key for the Cloud Services Connector.
 . Enable cloud services connectivity.
 
-=== Update OpenNMS Cloud Services Connector files
+=== Update Cloud Services Connector files
 
-The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to the OpenNMS-hosted cloud service.
+The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to the {page-component-title}-hosted cloud service.
```

Comment:
```suggestion
The OpenNMS Cloud Services Connector is an extension that lets a {page-component-title} instance send time series data to the OpenNMS-hosted cloud service.
```
This is referring to the Appliance Portal, but at the time of writing marketing suggested we not use the "Appliance Service" term. Technically it is The OpenNMS Group hosting the cloud service, but OpenNMS works in this context.